_posts/2015-12-09-foreman-community-newsletter-december-2015.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: Foreman Community Newsletter - December 2015
+author: Tomer Brisker
+tags:
+- foreman
+- katello
+- newsletter
+- openscap
+- xen
+- remote execution
+- high availability
+- windows imaging
+excerpt: |
+  Foreman 1.10 and Katello 2.4 coming soon
```

Comment:
This is quite dry - remember that people will read this before deciding to click through to the post. Maybe mention the new handbook, and the case study/deep dive updates?